### PR TITLE
SIRI-965: Moved methods from sirius-biz to sirius-web to reuse common methods

### PR DESCRIPTION
**Pull request by mku-sci**
### Additional Notes

- This PR fixes or works on following ticket(s): [SIRI-965](https://scireum.myjetbrains.com/youtrack/issue/SIRI-965)
- This PR is related to PR: https://github.com/scireum/sirius-biz/pull/1998

### Checklist

- [x] Code change has been tested and works locally
- [x] Code was formatted via IntelliJ and follows SonarLint & [best practices](https://scireum.myjetbrains.com/youtrack/articles/MISC-A-16/CodeStyle-JavaDoc)
- [ ] Patch Tasks: Is local execution of Patch Tasks necessary? If so, please also mark the PR with the tag.

### CODE REVIEW

**Review by jakobvogel — APPROVED** [review]

File: src/main/resources/default/assets/common/core.js.pasta [hunk 1]
```diff
@@ -816,6 +816,52 @@ sirius.extractFileExtension = function (filename) {
     return '.' + nameAndExtension.pop();
 }
 
+/**@
+ * Checks if the given file extension is a displayable image.
+ *
+ * @param extension the extension to check
+ * @returns {boolean} true if the extension is a displayable image, false otherwise
+ */
+
```
Comment (idlira) [inline src/main/resources/default/assets/common/core.js.pasta:825]: Remove this empty line.

File: src/main/resources/default/assets/common/core.js.pasta [hunk 2]
```diff
@@ -816,6 +816,52 @@ sirius.extractFileExtension = function (filename) {
...
+    return extension === '.jpg' || extension === '.jpeg' || extension === '.png' || extension === '.gif' || extension === '.webp';
+}
+
+/**@
+ * Determines the icon class for the given file extension.
+ *
+ * @param extension the extension to determine the icon for
+ * @returns {string} the icon class for the given extension
+ */
+
```
Comment (idlira) [inline src/main/resources/default/assets/common/core.js.pasta:836]: Remove this empty line.

**Review by idlira — APPROVED** [review]

**Review by scireum-mbo — APPROVED** [review]

File: src/main/resources/default/assets/common/core.js.pasta [hunk 3]
```diff
@@ -816,6 +816,52 @@ sirius.extractFileExtension = function (filename) {
     return '.' + nameAndExtension.pop();
 }
 
+/**@
+ * Checks if the given file extension is a displayable image.
+ *
+ * @param extension the extension to check
```
Comment (mko-sci) [inline src/main/resources/default/assets/common/core.js.pasta:822]: Would be nice if we would document the type of the extension on this occasion.

File: src/main/resources/default/assets/common/core.js.pasta [hunk 4]
```diff
@@ -816,6 +816,52 @@ sirius.extractFileExtension = function (filename) {
...
+ */
+
+sirius.isDisplayableImage = function (extension) {
+    return extension === '.jpg' || extension === '.jpeg' || extension === '.png' || extension === '.gif' || extension === '.webp';
+}
+
+/**@
+ * Determines the icon class for the given file extension.
+ *
+ * @param extension the extension to determine the icon for
```
Comment (mko-sci) [inline src/main/resources/default/assets/common/core.js.pasta:833]: Would be nice if we would document the type of the extension on this occasion.

**Review by mko-sci — APPROVED** [review]

File: src/main/resources/default/assets/common/core.js.pasta [hunk 5]
```diff
@@ -816,6 +816,52 @@ sirius.extractFileExtension = function (filename) {
...
+ * @returns {string} the icon class for the given extension
+ */
+
+sirius.determineFileIcon = function (extension) {
+    if (extension === '.pdf') {
+        return 'fa-file-pdf';
+    }
+
+    if (extension === '.tif' || extension === '.tiff' || extension === '.bmp' || extension === '.svg') {
```
[the remark this answers is not in the document]
Comment (sabieber) [reply src/main/resources/default/assets/common/core.js.pasta:842]: No it shouldnt

**Review by sabieber — APPROVED** [review]

File: src/main/resources/default/assets/common/core.js.pasta (hunk 3 above)
Comment (mko-sci) [reply src/main/resources/default/assets/common/core.js.pasta:822]: I meant it like this: `@param {string}  extension the extension to check`

File: src/main/resources/default/assets/common/core.js.pasta [hunk 6]
```diff
@@ -816,6 +816,50 @@ sirius.extractFileExtension = function (filename) {
     return '.' + nameAndExtension.pop();
 }
 
+/**@
+ * Checks if the given file extension corresponds to an image format that can be displayed in the frontend.
+ *
+ * @param extension the extension to check
```
Comment (mko-sci) [inline src/main/resources/default/assets/common/core.js.pasta:822]:
```suggestion
 * @param {string} extension the extension to check
```

**Review by mkeckmkeck — APPROVED** [review]

File: src/main/resources/default/assets/common/core.js.pasta (hunk 6 above)
Comment (mko-sci) [reply src/main/resources/default/assets/common/core.js.pasta:822]: What is with this one?

**Review by mko-sci — APPROVED** [review]

**Review by ymo-sci — APPROVED** [review]

**Pull request merged**